artemis-server/src/main/java/org/apache/activemq/artemis/core/config/HAPolicyConfiguration.java

```diff
@@ -21,14 +21,14 @@
 public interface HAPolicyConfiguration extends Serializable {
 
    enum TYPE {
-      LIVE_ONLY("Live Only"),
+      PRIMARY_ONLY("Primary Only"),
       REPLICATED("Replicated"),
       REPLICA("Replica"),
-      SHARED_STORE_MASTER("Shared Store Master"),
-      SHARED_STORE_SLAVE("Shared Store Slave"),
+      SHARED_STORE_PRIMARY("Shared Store Primary"),
+      SHARED_STORE_BACKUP("Shared Store Backup"),
       COLOCATED("Colocated"),
-      PRIMARY("Primary"),
-      BACKUP("Backup");
+      REPLICATION_PRIMARY("Replication Primary w/pluggable quorum voting"),
+      REPLICATION_BACKUP("Replication Backup w/pluggable quorum voting");
```

Comment:
I like your idea of using primary/backup terms for both shared store and pluggable quorum vote replication. What about to use primary/backup terms for the replication without pluggable quorum vote too?

Some suggestions:
```
REPLICATED("Replicated") -> REPLICATION_PRIMARY("Replication Primary")
REPLICA("Replica") -> REPLICATION_BACKUP("Replication Backup")
PRIMARY("Primary") -> COORDINATED_REPLICATION_PRIMARY("Coordinated Replication Primary")
BACKUP("Backup") -> COORDINATED_REPLICATION_BACKUP("Coordinated Replication Backup")
```

```
REPLICATED("Replicated") -> REPLICATION_PRIMARY("Replication Primary")
REPLICA("Replica") -> REPLICATION_BACKUP("Replication Backup")
PRIMARY("Primary") -> MANAGED_REPLICATION_PRIMARY("Managed Replication Primary")
BACKUP("Backup") -> MANAGED_REPLICATION_BACKUP("Managed Replication Backup")
```

```
REPLICATED("Replicated") -> REPLICATION_PRIMARY("Replication Primary")
REPLICA("Replica") -> REPLICATION_BACKUP("Replication Backup")
PRIMARY("Primary") -> REPLICATION_WITH_MANAGER_PRIMARY("Replication with Manager Primary")
BACKUP("Backup") -> REPLICATION_WITH_MANAGER_BACKUP("Replication with Manager Backup")
```